ui/src/app/base/components/FormikField/FormikField.tsx

```diff
@@ -4,12 +4,21 @@ import PropTypes from "prop-types";
 import React, { useRef } from "react";
 import { nanoid } from "@reduxjs/toolkit";
 
+import { TSFixMe } from "app/base/types";
+
+type Props = {
+  Component: JSX.Element;
+  name: string;
+  value: string;
+  [x: string]: TSFixMe;
+};
+
 const FormikField = ({
   component: Component = Input,
```

Comment:
Hmm seems like JSX needs the capital C component otherwise I get the error: `Property 'component' does not exist on type 'JSX.IntrinsicElements'`. An alternative could be changing the prop name to `Component` but I think leaving it as `component` keeps it in line with every other prop